pipeline/splitters.go

```diff
@@ -273,6 +274,46 @@ func (h *HekaFramingSplitter) UnframeRecord(framed []byte, pack *PipelinePack) [
 	return unframed
 }
 
+type MessagePackSplitter struct {
+	*MessagePackSplitterConfig
+	sr SplitterRunner
+}
+
+type MessagePackSplitterConfig struct {
+	UseMsgBytes bool `toml:"use_message_bytes"`
+}
+
+func (m *MessagePackSplitter) Init(config interface{}) error {
+	m.MessagePackSplitterConfig = config.(*MessagePackSplitterConfig)
+	return nil
+}
+
+func (m *MessagePackSplitter) SetSplitterRunner(sr SplitterRunner) {
+	m.sr = sr
+}
+
+func (m *MessagePackSplitter) ConfigStruct() interface{} {
+	return &MessagePackSplitterConfig{
+		UseMsgBytes: false,
```

Comment:
The UseMsgBytes config option is automatically supported by Heka for [every splitter](http://hekad.readthedocs.org/en/v0.10.0b1/config/splitters/index.html#common-splitter-parameters), and you're not changing the default, so I don't think there's any reason for you to have a UseMsgBytes option. Which in turn means you don't even need a custom config struct.